test/test.js

```diff
@@ -3,64 +3,76 @@ var assert = require('assert');
 var getWindow = require('../');
 var getDocument = require('get-document');
 
+var inNode = require('detect-node');
+
+// Zuul is configured to pass on to Browserify a configuration to exclude the
+// following module, 'jsdom' because it is only for testing in node. Browsers
+// have real dom
+var jsdom = inNode ? require('jsdom') : null;
+
+var doc = inNode ? jsdom.jsdom() : document;
+var win = inNode ? doc.parentWindow : window;
+
+
 describe('get-window', function () {
 
   it('should work with the global Window object', function () {
-    var win = getWindow(window);
-    assert(win === window);
+    var result = getWindow(win);
+    assert(result === win);
   });
 
   it('should work with the global Document object', function () {
-    var win = getWindow(document);
-    assert(win === window);
+    var result = getWindow(doc);
+    assert(result === win);
   });
 
   it('should work with the `document.body` object', function () {
-    var win = getWindow(document.body);
-    assert(win === window);
+    var result = getWindow(doc.body);
+    assert(result === win);
   });
 
   it('should work with the a DOM element inside `document.body`', function () {
-    var win = getWindow(document.body.firstChild);
-    assert(win === window);
+    doc.body.appendChild(doc.createElement("p"));
```

Comment:
Well perhaps, but we're not cleaning them up afterwards by removing them from the DOM. The tests should be idempotent IMO.